beta-20170406.md

```diff
@@ -0,0 +1,72 @@
+---
+title: What&#39;s New in beta-20170406
+toc: false
+summary: Additions and changes in CockroachDB version beta-20170406.
+---
+
+## Apr 6, 2017
+
+Get future release notes emailed to you:
+<div class="hubspot-install-form install-form-1 clearfix">
+    <script>
+        hbspt.forms.create({
+            css: '',
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+### Binaries
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-beta-20170406.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-beta-20170406.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+</div>
+
+### Backwards-Incompatible Changes
+
+- The implementation of indexing for collated string columns has changed. Any indexes containing collated strings must be dropped and recreated. [#14528](https://github.com/cockroachdb/cockroach/pull/14528)
```

Comment:
May be good to note here that collated strings in primary keys are fine (i.e., only secondary indexes are affected).

We also need to note that any row with `-0` float in a primary key or index needs to be deleted before upgrading and then added back after upgrading. Otherwise the `-0` row will show up during table scans, but will not be able to be deleted or selected as part of an index scan.